tests/fertiscan/db/update_inspection/test_update_sub_labels.py

```diff
@@ -206,8 +207,84 @@ def test_update_sub_labels(self):
         for expected_updated_item in expected_updated_data:
             self.assertTrue(
                 expected_updated_item in updated_data,
-                f"Expected updated item {expected_updated_item} was not found in the updated data"
+                f"Expected updated item {expected_updated_item} was not found in the updated data",
+            )
+
+    def test_update_sub_labels_with_mismatched_arrays(self):
+        # Create sub labels with mismatched 'fr' and 'en' arrays
+        mismatched_sub_labels = json.dumps(
```

Comment:
I suggest you use the dict from your set up and remove one enrty from instructrion. This way makes the files lighter to read and easier to maintain

Reply:
Done.